modules/fast_double_parser/metadata.json

```diff
@@ -0,0 +1,16 @@
+{
+    "homepage": "https://github.com/lemire/fast_double_parser",
+    "maintainers": [
+        {
+            "email": "bcr-maintainers@bazel.build",
```

Comment:
would you mind listing yourself as the maintainer here? it should be very low effort :)

Reply:
Okay.